recipes/mgltools/meta.yaml

```diff
@@ -0,0 +1,25 @@
+about:
+    home: http://mgltools.scripps.edu/
+    license_file: LICENSES
+    summary: MGLTools is an program for visualization and analisys of molecular structures.
+package:
+    name: mgltools
+    version: 1.5.6
+build:
+  number: 0
+  skip: False
+source:
+    fn:  mgltools_x86_64Linux2_1.5.6.tar.gz                                                                             # [linux]
+    url: http://mgltools.scripps.edu/downloads/downloads/tars/releases/REL1.5.6/mgltools_x86_64Linux2_1.5.6.tar.gz      # [linux]
```

Comment:
Hi @bgruening , about the template syntax, I made it.

``` yml
source:
fn:  mgltools_x86_64Linux2_1.5.6.tar.gz # [linux]
url: http://mgltools.scripps.edu/downloads/downloads/tars/releases/REL1.5.6/mgltools_x86_64Linux2_1.5.6.tar.gz # [linux]
md5: e3f62852c4d442a292dedbe6c79540ca # [linux]
fn:  mgltools_i86Darwin9_1.5.6.tar.gz  # [osx]
url: http://mgltools.scripps.edu/downloads/downloads/tars/releases/REL1.5.6/mgltools_i86Darwin9_1.5.6.tar.gz # [osx]
md5: c515b187cddeec72fe5a81ecb4a92d6b # [osx]
```
There are anything wrong?
This is a binary distribution. Thanks for your suggestions! :)